### DISCUSSION TITLE
Adding Herten to Germany

### DISCUSSION THREAD

**Discussion by vmhomelab** [mampfes/hacs_waste_collection_schedule#2064] [Ideas]
You have already added Recklinghausen, Datteln Oer-Erkenschwick, why not add Herten too? :)

**Comment by 5ila5** [comment]
Herten is supported by the [ICS source ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) I added dedicated documentation in #2071 you can find it [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/herten_de.md)
